wdn/templates_4.1/less/layouts/share.less

```diff
@@ -183,6 +183,18 @@
     }
 }
 
+/* the share widget dropdown needs some focus love because it's icon is absolutely positioned */
+.wdn-dropdown-widget-label.focused .wdn-icon-share {
+	/* firefox */
+	outline: 1px dotted #212121;
+	/* -webkit-focus-ring-color = '#5B9DD9' */
```

Comment:
Ah, webkit has a different default outline color/style. This was an attempt to simulate it if it existed. I will be honest, this was a copy/paste from stack overflow. The comment is simply stating what the value of `-webkit-focus-ring-color` should be. If you want, the comment can be removed as it isn't super helpful.